test_package/CMakeLists.txt

```diff
@@ -8,6 +8,10 @@ conan_basic_setup()
 
 file(GLOB SOURCE_FILES *.cpp)
 
+if(WITH_GMOCK)
+    add_definitions("-DWITH_GMOCK")
```

Comment:
We should probably conditionally declare this from `package_info` int he package no?

Reply:
Normally that would be correct, but this is not a gmock/gtest manipulation flag. It has been specifically created for the test of this conan package from what I see (I did not create it myself). It is only used in test_package/test_package.cpp to activate/deactivate the part of the code that tests gmock, depending on whether the package has been built with gmock or not.